README.md

```diff
@@ -9,13 +9,15 @@ A starter repository showing how to build a blog with the [Eleventy](https://git
 * [Netlify](https://eleventy-base-blog.netlify.com/)
 * [GitHub Pages](https://11ty.github.io/eleventy-base-blog/)
 * [Remix on Glitch](https://glitch.com/~11ty-eleventy-base-blog)
+* [Azure Static Web Apps](https://aka.ms/11ty-swa)
```

Comment:
Specifically, this link goes directly to the demo of the live deployed site on the service.